ProjectManager.cs

```diff
@@ -4,40 +4,66 @@
 using System.Runtime.InteropServices;
 using System.Text;
 using System.Threading.Tasks;
+using TrainingApp2.Projects;
 
 namespace TrainingApp2
 {
     public class ProjectManager
     {
-        List<Project> projectsList = new List<Project>();
-        public void AddProject(string name, string description, DateTime startTime, DateTime endTime)
+        public int limitNumberOfProjects = 0;
```

Comment:
To wymaga udoskonalenia. Chcialbym zebys przeczytac o enumach w ksiazce ktora Ci pozyczylem